internal/batches/ui/tui.go

```diff
@@ -449,3 +453,10 @@ func batchCompletePending(p output.Pending, message string) {
 func batchCompleteWarning(p output.Pending, message string) {
 	p.Complete(output.Line(batchWarningEmoji, batchWarningColor, message))
 }
+
+func dockerWatchDogWarning(out *output.Output, err error) {
+	block := out.Block(output.Line("🐳", output.StyleWarning, "Hey there, Your docker engine might be frozen."))
+	block.WriteLine(output.Line("", output.StyleWarning, "If there's no progress in the next couple minutes, you can try rebooting docker and running the command again."))
```

Comment:
Very slight wording tweaks (more input encouraged from other Batchers!):

```suggestion
	block := out.Block(output.Line("🐳", output.StyleWarning, "Hey there, your Docker engine might be frozen."))
	block.WriteLine(output.Line("", output.StyleWarning, "If there's no progress in the next couple minutes, you may want to try restarting Docker and running the command again."))
```

Reply:
This looks good.

@danielmarquespt Do you have any thoughts?

Reply:
Personal feeling, I would remove the `"Hey there"`, and just leave it as `"It seems like your docker engine might be frozen..."`

Reply:
Got it. Updated.